care/facility/models/patient_consultation.py

```diff
@@ -238,6 +238,8 @@ class PatientConsultation(PatientBaseModel, ConsultationRelatedPermissionMixin):
     prn_prescription = JSONField(default=dict)
     discharge_advice = JSONField(default=dict)
 
+    consent_records = JSONField(default=list)
```

Comment:
I think having a fk to the consent record file upload would be better

Reply:
there are multiple types of consent records, and multiple files can be uploaded for each. This one just keeps track of which types have been selected. I did not add an FK as the file upload is already linked to the consultation

Reply:
okay